wesgi/tests.py

```diff
@@ -31,7 +30,11 @@ def Response(status=200, headers=None):
 
 def run_mw(mw):
     start_response = Mock()
+
     request = webob.Request.blank("")
+    if mw.policy.forward_headers:
+        request.headers['Content-Type'] = 'text/xml'
+
```

Comment:
could you rather pass in the request headers as a keyword argument to run_mw()? That way the test decides what headers to pass.